elliott/elliottlib/cli/snapshot_cli.py

```diff
@@ -237,7 +263,6 @@ def snapshot_cli():
     metavar='PATH',
     help='Path to the pull secret file to use. For example, if the images are in quay.io/org/repo then provide the pull secret to read from that repo.',
 )
-@click.option('--for-fbc', is_flag=True, help='To indicate that the given builds are fbc builds.')
```

Comment:
thanks for removing this. Can you also remove it from konflux_release_cli.py as well? I added it there so i can pass it on here